src/infrastructure/dataSchemes/plotDataScheme.ts

```diff
@@ -0,0 +1,35 @@
+//NOTE: Includes the actual data collected from a figure: axis values, axis scale, data point values and so
+//NOTE: Do not include any data which depends on StarryDigitizer app(e.g. setting axis by 2 points or 4 points etc...)
+
+export interface PlotDataScheme {
+  schemeVersion: '0.1.0'
+  axisSets: {
+    id: number
+    name: string
+    xAxis: {
+      value1: number
+      value2: number
```

Comment:
minValue and maxValue should be clearer, but following the current implementation of StarryDigitizer, that the x1 value can be larger than x2 value